### DISCUSSION TITLE
mol_from_text: convert ASCII into structures!

### DISCUSSION THREAD

**Discussion by prheenan** [rdkit/rdkit#8102] [Show and tell] [answered]
Inspired by the [NanoPutian](https://en.wikipedia.org/wiki/NanoPutian) and its applications to science education, I've made a "just-for-fun" repository based on rdkit which is called [mol_from_text](https://github.com/prheenan/mol_from_text) which converts text into molecular structures, with optional animations. Posting here in case anyone else wants to take it for a spin! All feedback welcome.

**Comment by greglandrum** [comment, marked as answer]
Nice!
